### PR TITLE
Added necessary image and button for Landing Page

### PR DESCRIPTION
**Pull request by satyam5340**

### CODE REVIEW

File: client/src/Components/HomePage/HomePage.js [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-				</p>
-			</header>
+class HomePage extends React.Component{
+	
+	render(){
+		return <div class='entry-main'>
+			<div className='entry-container'>
+				<img src={mainImage} />
+				<div className='entry-name'>
+					<p className='entry-title'>10x Team G</p>
```
Comment (rahulbaroqoe) [inline client/src/Components/HomePage/HomePage.js]: It should be 10x Team 7

File: routes/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 const express = require("express");
 const router = express.Router();
 
+
 /* GET home page. */
 router.get("/", function (req, res, next) {
-	res.render("index", { title: "Express" });
+	res.render("satyam", { title: "Leenesh"});
```
Comment (rahulbaroqoe) [inline routes/index.js:7]: Is this mistakenly checked in?

**Review by rahulbaroqoe — CHANGES_REQUESTED** [review]

File: client/src/Components/HomePage/HomePage.js (hunk 1 above)
Comment (satyam5340) [reply client/src/Components/HomePage/HomePage.js]: I will change that asap

File: routes/index.js (hunk 2 above)
Comment (satyam5340) [reply routes/index.js:7]: Yes sir i was telling the team members how to do a pull request and add reviewers so i pushed this one

**Review by VishhMj — APPROVED** [review]

**Review by premkalyan-keerthi — APPROVED** [review]

**Pull request merged**